hq/app/logic/InspectorResults.scala

```diff
@@ -0,0 +1,104 @@
+package logic
+
+import com.amazonaws.services.inspector.model.{AssessmentRun, DescribeAssessmentRunsResult, _}
+import model.InspectorAssessmentRun
+import org.joda.time.DateTime
+import utils.attempt.FailedAttempt
+
+import scala.collection.JavaConverters._
+
+
+object InspectorResults {
+  private val tagMatch = "[\\w\\-_\\.]"
+  val RunNameMatch = s"AWSInspection-($tagMatch+)-($tagMatch+)-($tagMatch+)-[\\d]+".r
```

Comment:
This will change if the Lambda changes its format

Reply:
+1

Reply:
We can change this separately and exclude non-Lambda runs at the same time